hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/procedure/TestBootstrapProcedure.scala

```diff
@@ -93,6 +94,8 @@ class TestBootstrapProcedure extends HoodieSparkProcedureTestBase {
       spark.sql("set hoodie.datasource.write.row.writer.enable = false")
       spark.sql(s"""call run_clustering(table => '$tableName')""".stripMargin)
       assertResult(0)(spark.sql(s"select * from $tableName").except(beforeClusterDf).count())
+
+      spark.sessionState.conf.unsetConf("unset hoodie.metadata.index.column.stats.enable")
```

Comment:
[HUDI-8774](https://issues.apache.org/jira/browse/HUDI-8774) track bootstrap support for col stats